server/prisma/generator/factories/chapterRoles.factory.ts

```diff
@@ -4,18 +4,19 @@ const chapterPermissions = [
   'chapter-edit',
   'event-create',
   'event-edit',
+  'rsvp',
 ] as const;
 
 type Permissions = typeof chapterPermissions[number];
 
 const roles: Array<{ name: string; permissions: Permissions[] }> = [
   {
     name: 'administrator',
-    permissions: ['chapter-edit', 'event-create', 'event-edit'],
+    permissions: ['chapter-edit', 'event-create', 'event-edit', 'rsvp'],
   },
   { name: 'organizer', permissions: ['event-create', 'event-edit'] },
   { name: 'moderator', permissions: ['event-edit'] },
```

Comment:
I guess these two also should get `rsvp` permission.

Reply:
Yeah, post MVP, but for now I think we can get rid of these roles and (try to) keep things reasonably simple.